airbyte-cdk/python/airbyte_cdk/sources/declarative/yaml_declarative_source.py

```diff
@@ -21,11 +23,17 @@ def __init__(self, path_to_yaml, debug: bool = False):
         super().__init__(source_config, debug)
 
     def _read_and_parse_yaml_file(self, path_to_yaml_file) -> ConnectionDefinition:
-        package = self.__class__.__module__.split(".")[0]
-
-        yaml_config = pkgutil.get_data(package, path_to_yaml_file)
-        decoded_yaml = yaml_config.decode()
-        return self._parse(decoded_yaml)
+        try:
```

Comment:
I didn't find a way to just pass a file path to a component in the airbyte_cdk to create a source so I had to add this. Let me know if I missed something